.NET/Microsoft.Recognizers.Text.DataTypes.DataDrivenTests/TestTimexResolver.cs

```diff
@@ -533,7 +533,7 @@ public void DataTypes_Resolver_DateRange_Decimal_Period_PT()
         public void DataTypes_Resolver_DateRange_Demaical_Period_EN()
         {
             var sourceLanguage = CultureInfo.CurrentCulture;
-            var testLanguage = new CultureInfo("en-En", false);
+            var testLanguage = new CultureInfo("en-US", false);
```

Comment:
Is there any specific reason for this change?

Reply:
"en-EN" is not always recognized because it is not a supported identifier according to this [list](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c?redirectedfrom=MSDN) (I was getting an error when running the test).